compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -475,6 +480,10 @@ internal class RootNodeOwner(
                 if (!measureAndLayoutDelegate.hasPendingMeasureOrLayout) {
                     measureAndLayoutDelegate.dispatchOnPositionedCallbacks()
                 }
+                @OptIn(ExperimentalComposeUiApi::class)
+                if (ComposeUiFlags.isRectTrackingEnabled) {
+                    rectManager.dispatchCallbacks()
```

Comment:
In the latest [AOSP](https://raw.githubusercontent.com/androidx/androidx/refs/heads/androidx-main/compose/ui/ui/src/androidMain/kotlin/androidx/compose/ui/platform/AndroidComposeView.android.kt) this is also called on:
- onLayout -> updatePositionCacheAndDispatch (after the Compose layout, before draw)
- dispatchDraw (after the Compose draw)

It seems we also need to call it? The similar places in our code are `fun measureAndLayout()` and ` fun draw(canvas: Canvas)` in this file.

P.S. The `measureAndLayout(layoutNode: LayoutNode, constraints: Constraints)` is called not on every layout, only when some widget (`LazyList`, for example) requests to re-layout some particular node.

Reply:
> In the latest [AOSP](https://raw.githubusercontent.com/androidx/androidx/refs/heads/androidx-main/compose/ui/ui/src/androidMain/kotlin/androidx/compose/ui/platform/AndroidComposeView.android.kt)

Please make attention on 1.8 vs 1.9 diff here during backport to 1.8.2

Reply:
Created https://youtrack.jetbrains.com/issue/CMP-8183/Support-calling-of-RectManager.updateOffsets, there is a new potential issue

Reply:
>The similar places in our code are fun measureAndLayout() and  fun draw(canvas: Canvas) in this file.

Added